python/rmm/_lib/lib.pyx

```diff
@@ -269,3 +269,37 @@ def rmm_getallocationoffset(ptr, stream):
     result = int(c_offset[0])
     free(c_offset)
     return result
+
+
+cdef memory_pair c_getinfo(
+    cudaStream_t stream
+) except *:
+    """Get total and free memory sizes using the RMM memory manager by calling
+    the librmm functions via Cython
+    """
+    cdef size_t free
+    cdef size_t total
+
+    with nogil:
+        rmm_error = rmmGetInfo(
+            <size_t*>&free,
+            <size_t*>&total,
+            <cudaStream_t>stream
+        )
+
+    check_error(rmm_error)
+
+    return memory_pair(free, total)
+
+
+def rmm_getinfo(stream):
+    """Get total and free memory sizes using the RMM memory manager by calling
+    the librmm functions via Cython
+    """
+    cdef cudaStream_t c_stream = <cudaStream_t><size_t>stream
```

Comment:
This was recently discussed and we determined that a cudaStream is a pointer underneath and that it's more appropriate to use `uintptr_t` as compared to `size_t`.
```suggestion
    cdef cudaStream_t c_stream = <cudaStream_t><uintptr_t>stream
```

Reply:
Thanks! I've made this change.